configure.ac

```diff
@@ -345,7 +345,7 @@ AC_SUBST([MPI_FCLIBS])
 AC_SUBST([MPI_FLIBS])
 
 [EXTRA_CFLAGS="$EXTRA_CFLAGS $OPENMP_CXXFLAGS -std=c99 -msse4.2"]
-[EXTRA_CXXFLAGS="$EXTRA_CXXFLAGS $OPENMP_CXXFLAGS -std=c++11 -msse4.2"]
+[EXTRA_CXXFLAGS="$EXTRA_CXXFLAGS $OPENMP_CXXFLAGS -std=c++17 -msse4.2"]
```

Comment:
✨
I guess we can start doing some things differently after this. First things that come to mind are things we've pulled into the geopm namespace, like std::make_unique and std::hardware_destructive_interference_size.

Reply:
Yeah, we should open an issue to deprecate their use.

Reply:
https://github.com/geopm/geopm/issues/2827